flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -221,9 +222,10 @@ class _SingleTabState extends State<SingleTab> {
   }
 
   @override
-  void dispose() {
-    _controller.stopRendererScanning();
-    _controller.removeListener(() {});
+  void dispose() async {
     super.dispose();
+    _controller.removeListener(() {});
```

Comment:
This line does exactly nothing.
We can remove a listener after we've added one, and using exactly the same reference that we used when adding it.

We should also remove a similar line in `README.md` and in `multiple_tab.dart`